docs/architecture/ADR-003-borrow-assets.md

```diff
@@ -47,6 +47,29 @@ The calculated borrow limit, which weighs collateral uTokens against borrowed as
 
 Note also that as a consequence of uToken interest, the asset value of uToken collateral increases over time, meaning a user who repays positions in full and redeems collateral uTokens will receive back more base assets than they deposited originally, reducing the effective interest.
 
+### Max collateral utilization
+
+We define a _collateral utilization_ as ratio of lended collateral to total amount of collateral per token. Collateral utilization is 0 when there is no borrow of given token collateral. It is equal to 1, when all token collateral is borrowed. Collateral utilization of token X is growing when lenders withdraw their token X collateral or borrowers take a new loan of token X.
```

Comment:
Need clarification. Imagine 3 users:

- Alice lends `100 ATOM`, does not enable atom as collateral
- Bob lends `200 ATOM`, enables atom as collateral
- Charlie borrows `100 ATOM`

In this case, `borrow utilization` is `100/300`. (Equal to the amounts `borrowed / lent`)

Would collateral utilization be the same? Or would it be `100/100` (equal to `borrowed / collateral`).

If we desire the first option, we should stick to the existing concept `Borrow Utilization` and avoid using the word "collateral" to refer to ALL assets lent, whether enabled or disabled as collateral.